orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/loader/v2/V2TemplateLoader.java

```diff
@@ -16,48 +16,38 @@
 
 package com.netflix.spinnaker.orca.pipelinetemplate.loader.v2;
 
-import com.netflix.spinnaker.orca.pipelinetemplate.exceptions.TemplateLoaderException;
-import com.netflix.spinnaker.orca.pipelinetemplate.v1schema.model.TemplateConfiguration;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import com.netflix.spinnaker.orca.clouddriver.OortService;
 import com.netflix.spinnaker.orca.pipelinetemplate.v2schema.model.V2PipelineTemplate;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
+import retrofit.client.Response;
 
-import java.net.URI;
-import java.net.URISyntaxException;
-import java.util.Collection;
-
-import static java.lang.String.format;
+import java.io.IOException;
 
 @Component
 public class V2TemplateLoader {
-  private Collection<V2TemplateSchemeLoader> schemeLoaders;
 
-  // TODO(jacobkiefer): Use Artifact resolution instead of custom template loaders.
-  @Autowired
-  public V2TemplateLoader(Collection<V2TemplateSchemeLoader> schemeLoaders) {
-    this.schemeLoaders = schemeLoaders;
-  }
+  private OortService oortService;
+
+  private ObjectMapper objectMapper;
 
-  /**
-   * @return a LIFO list of pipeline templates
-   */
-  public V2PipelineTemplate load(TemplateConfiguration.TemplateSource source) {
-    return load(source.getSource());
+  public V2TemplateLoader() {}
```

Comment:
Why do we need the no-args constructor?  It looks like the resulting `V2TemplateLoader` would just throw NPEs.  Along the same lines, could we set the `oortService` and `objectMapper` to `final`?

Reply:
Removed the noargs constructor. I don't think the `final` is necessary.